crates/swc_ecma_minifier/tests/fixture/issues/2257/full/input.js

```diff
@@ -2624,7 +2624,7 @@
                     return /* reexport */ _toPropertyKey;
                 },
                 typeOf: function () {
-                    return /* reexport */ _type_of_typeof;
+                    return /* reexport */ _typeof_typeof;
```

Comment:
Can you revert input files of tests, please?

Reply:
When I execute `cargo test` with the flag `UPDATE=1`, the input file is automatically updated.
I am currently investigating the cause.

Reply:
never mind. The the It's `_type_of_typeof`. what a confusing name.